src/skmultiflow/data/data_stream.py

```diff
@@ -403,7 +403,7 @@ def get_data_info(self):
         if self.task_type == self._CLASSIFICATION:
             return "{}{} target(s), {} classes".format(name, self.n_targets, self.n_classes)
         elif self.task_type == self._REGRESSION:
-            return "{} target(s)".format(name, self.n_targets)
+            return "{}{} target(s)".format(name, self.n_targets)
```

Comment:
Don't we need some space here between the two **{}**?

Reply:
Hi @garawalid, thanks for pointing it out! I just checked the `name` variable definition:

https://github.com/scikit-multiflow/scikit-multiflow/blob/9c4168e37d0f638a698d0af2916aba866e83dbe8/src/skmultiflow/data/data_stream.py#L402

It already adds a space in case `self.name` is not `None` (`None` is the default value during the object creation).

Reply:
Great!